packages/react/src/Dialog/Dialog.tsx

```diff
@@ -17,7 +18,9 @@ export const Dialog = forwardRef(
     <dialog {...restProps} ref={ref} className={clsx('ams-dialog', className)}>
       <form method="dialog" className="ams-dialog__form">
         <header className="ams-dialog__header">
-          <span className="ams-dialog__title">{title}</span>
+          <Heading level={1} size="level-4">
+            {title}
+          </Heading>
           <IconButton label="Sluiten" formNoValidate />
```

Comment:
If the heading is now level 4, shouldn't the close button be as well?

Reply:
Yes, adjusted.